model-optimizer/mo/utils/versions_checker.py

```diff
@@ -82,14 +86,28 @@ def parse_and_filter_versions_list(required_fw_versions, version_list, env_setup
                 # this python_version requirement is not satisfied to required environment
                 # and requirement for a dependency will be skipped
                 return version_list
+        elif env_marker == 'sys_platform' and env_marker in env_setup:
+            splited_env_req[1] = splited_env_req[1].strip(' ').replace('\'', '')
```

Comment:
You support only single quotes to specify the platform name. But what if the platform name is wrapped with double quotes? This will result in an error in parser:
```
TypeError: '<' not supported between instances of 'str' and 'int'
```

Please, add unit test for that.

Reply:
Actually, similar comment for the "python_version". It support only double quotes but not single quotes. Add unit tests for that as well.

Reply:
Now both `python_version` and `sys_platform` handled with single and double quotes. Also added unit tests for that cases.